common/lib/xmodule/xmodule/tests/test_content.py

```diff
@@ -95,9 +101,9 @@ def test_thumbnail_none(self):
         self.assertIsNone(content.thumbnail_location)
 
     def test_static_url_generation_from_courseid(self):
-        course_key = SlashSeparatedCourseKey('foo', 'bar', 'bz')
+        course_key = SlashSeparatedCourseKey('a', 'b', 'bz')
         url = StaticContent.convert_legacy_static_url_with_course_id('images_course_image.jpg', course_key)
-        self.assertEqual(url, '/c4x/foo/bar/asset/images_course_image.jpg')
+        self.assertEqual(url, '/c4x/a/b/asset/images_course_image.jpg')
```

Comment:
Might want to also try a URL that would test whether URL encoding/decoding behavior works properly. An image with a space or some other escaped char in the filename?